src/overlays/actors/ovl_En_Bee/z_en_bee.c

```diff
@@ -118,121 +118,124 @@ void EnBee_Destroy(Actor* thisx, PlayState* play) {
     Collider_DestroyCylinder(play, &this->collider);
 }
 
-void func_80B5A854(EnBee* this) {
+void EnBee_SetupIdle(EnBee* this) {
     s32 pad;
-    Vec3f sp48;
-    s16 sp46;
+    Vec3f tmpPos;
+    s16 yawOffset;
 
-    Animation_Change(&this->skelAnime, &object_bee_Anim_00005C, 1.0f, 0.0f,
-                     Animation_GetLastFrame(&object_bee_Anim_00005C), 0, -10.0f);
-    Math_Vec3f_Copy(&sp48, &this->actor.home.pos);
+    Animation_Change(&this->skelAnime, &gBeeFlyingAnim, 1.0f, 0.0f, Animation_GetLastFrame(&gBeeFlyingAnim), 0, -10.0f);
+    Math_Vec3f_Copy(&tmpPos, &this->actor.home.pos);
 
-    sp46 = (this->unk_218 * 0x700) + 0x2000;
-    sp48.x += Math_SinS(sp46) * 50.0f;
-    sp48.y = Rand_ZeroFloat(50.0f) + (this->actor.floorHeight + 30.0f);
-    sp48.z += Math_CosS(sp46) * 50.0f;
+    yawOffset = (this->instanceId * 0x700) + 0x2000;
+    tmpPos.x += Math_SinS(yawOffset) * 50.0f;
+    tmpPos.y = Rand_ZeroFloat(50.0f) + (this->actor.floorHeight + 30.0f);
+    tmpPos.z += Math_CosS(yawOffset) * 50.0f;
 
-    Math_Vec3f_Copy(&this->unk_21C[0], &sp48);
-    Math_Vec3f_Copy(&sp48, &this->actor.home.pos);
+    Math_Vec3f_Copy(&this->moveToDestinations[0], &tmpPos);
```

Comment:
I think we'd usually call this sort of thing a `targetPos` or similar.

Reply:
resolved in c61b019, changed `moveToDestinations` to `targetPos`, and related things like `nextDestination` and `destinationIndex` to `nextPos` / `posIndex`